samples.go

```diff
@@ -0,0 +1,27 @@
+package lft
+
+import (
+	metrics "github.com/rcrowley/go-metrics"
+	sample "github.com/remerge/go-lock_free_timer/sample"
+)
+
+func NewLockFreeSample(reservoirSize int) metrics.Sample {
+	// These buckets might be pulled out of a hat but are ok for now as a first approximation.
+	// 9435590846921020 is a max possible metrics's value in Prometheus on the day this code is written and
+	// was split to 8 equal ranges. Subject for a further tuning.
+	weirdDefaultBuckets := []float64{
```

Comment:
since we do not have clear (or request by anyone) ranges for buckets. we decided to go the way:
- Take the max possible existing value in `P.` (of any metric)
- Take the min possible existing value in `P.` (of any metric)
- Split the range `min...max` to N parts and use them as bounds for buckets.